### PR TITLE
dev-libs/capstone: add 6.0.0_alpha2, drop 6.0.0_alpha1, drop 5.0.1

### PR DESCRIPTION
**Pull request by hamarituc**
version bump

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: dev-libs/capstone/capstone-6.0.0_alpha2.ebuild [hunk 1]
```diff
@@ -74,7 +74,8 @@ src_prepare() {
 
 src_configure() {
 	local mycmakeargs=(
-		-DBUILD_SHARED_LIBS=true
+		-DCAPSTONE_BUILD_SHARED_LIBS=true
+		-DCAPSTONE_BUILD_STATIC_LIBS=false
 		-DCAPSTONE_BUILD_STATIC_RUNTIME=false
```
Comment (eli-schwartz) [inline dev-libs/capstone/capstone-6.0.0_alpha2.ebuild:79]:
```
 * QA Notice: One or more CMake variables were not used by the project:
 *   CAPSTONE_BUILD_STATIC_RUNTIME
```
It seems they removed both old variables at the same time.

**Review by eli-schwartz — CHANGES_REQUESTED** [review]
When upgrading to alpha1, you dropped `-DCAPSTONE_BUILD_TESTS="$(usex test)"` although I wonder whether maybe you should be defining `CAPSTONE_BUILD_CSTEST` instead.

As it is, the USE=test is unused so should have been removed after the only `usex` was removed.

**Comment by hamarituc** [comment]
> When upgrading to alpha1, you dropped `-DCAPSTONE_BUILD_TESTS="$(usex test)"` although I wonder whether maybe you should be defining `CAPSTONE_BUILD_CSTEST` instead.

`cstest` would require `libcyaml` which is unfortunately not available in Gentoo.

> As it is, the USE=test is unused so should have been removed after the only `usex` was removed.

I fixed it.

File: dev-libs/capstone/capstone-6.0.0_alpha2.ebuild (hunk 1 above)
Comment (hamarituc) [reply dev-libs/capstone/capstone-6.0.0_alpha2.ebuild]: fixed

**Comment by eli-schwartz** [comment]
> `cstest` would require `libcyaml` which is unfortunately not available in Gentoo.

Perhaps we could package it?

**Pull request closed without merging**